src/js/test/rpc/server.test.ts

```diff
@@ -196,59 +197,6 @@ describe("Server", function () {
         });
     });
 
-    it(
```

Comment:
I removed this test, because the new one does the same, with better implementation.